.dockerignore

```diff
@@ -0,0 +1,4 @@
+**/node_modules
+**/build
+**/logs
+**/.git
```

Comment:
```suggestion
**/.git

```

Reply:
GitHub complaining about no newline at end of file so might as well change it I guess (or not... I don't really care)